lib/src/precompute_client.dart

```diff
@@ -18,7 +18,7 @@ class SdkOptions {
   final String sdkKey;
 
   /// Platform for the SDK
-  final SdkPlatform sdkPlatform;
+  final SdkPlatform? sdkPlatform;
```

Comment:
made this optional - if users want to identify the platform that is appreciated. defaults to `dart-unknown`